apps/patient/src/views/Pharmacy.tsx

```diff
@@ -99,90 +101,124 @@ export const Pharmacy = () => {
     lat: number | undefined
     lng: number | undefined
```

Comment:
a lil out of scope, but this can be abbreviated
```
{ loc, lat, lng}: {
    loc?: string;
    lat?: number;
    lng?: number;
  }
```

Reply:
i'll take as improvement for when I'm back in there